pkg/tarbackfill/tarbackfill.go

```diff
@@ -0,0 +1,159 @@
+package tarbackfill
```

Comment:
Should this be a public c/storage API, or would something like `c/storage/internal/tarbackfill` work?

(I don’t have a strong opinion, I just want this to be an intentional decision.)